csharp/src/Google.Protobuf/Google.Protobuf.csproj

```diff
@@ -22,15 +22,33 @@
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'net45' or '$(TargetFramework)' == 'netstandard2.0' ">
-    <DefineConstants>$(DefineConstants);GOOGLE_PROTOBUF_SUPPORT_SYSTEM_MEMORY</DefineConstants>
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net45' ">
+    <DefineConstants>$(DefineConstants);GOOGLE_PROTOBUF_SUPPORT_SYSTEM_MEMORY;GOOGLE_PROTOBUF_SUPPORT_SPAN</DefineConstants>
```

Comment:
Something to discuss (we don't have to make the change yet):
with the current setup, it looks like GOOGLE_PROTOBUF_SUPPORT_SPAN is being used in exactly the same configurations as already existing define GOOGLE_PROTOBUF_SUPPORT_SYSTEM_MEMORY.

My feeling is that GOOGLE_PROTOBUF_SUPPORT_SPAN doesn't really make sense on its own and doesn't have a well-defined meaning (you cannot really "support span" if you don't also allow System.Memory dependency), so perhaps there's no use of introducing another define if we can already use GOOGLE_PROTOBUF_SUPPORT_SYSTEM_MEMORY?  I feel like  having too many defines harms readability.

Reply:
Now that IBufferMessage is supported on net45 then they're the same.

I'll replace GOOGLE_PROTOBUF_SUPPORT_SPAN with GOOGLE_PROTOBUF_SUPPORT_SYSTEM_MEMORY